tests/jballerina-integration-test/src/test/resources/testng.xml

```diff
@@ -269,8 +269,8 @@
     <test name="ballerina-observability-tests" parallel="false">
         <parameter name="enableJBallerinaTests" value="true"/>
         <classes>
-            <class name="org.ballerinalang.test.observability.tracing.TracingTestCase"/>
-            <!--<class name="org.ballerinalang.test.observability.metrics.MetricsTestCase"/>-->
+            <!--<class name="org.ballerinalang.test.observability.tracing.TracingTestCase"/>-->
```

Comment:
Lets add a git issue for enable this test case as well.

Reply:
Created https://github.com/ballerina-platform/ballerina-lang/issues/18348